frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -0,0 +1,39 @@
+import { timeouts } from '../utils/timeouts';
+
+export const listPage = {
+  titleShouldBe: (title) => cy.getElemByDataId('resource-title').should('have.text', title),
+  projectDropdownShouldExist: () => cy.getElemByDataId('namespace-bar-dropdown').should('exist'),
+  projectDropdownShouldNotExist: () =>
+    cy.getElemByDataId('namespace-bar-dropdown').should('not.exist'),
+  clickCreateYAMLButton: () => cy.getElemByDataId('yaml-create').click(),
+  filter: {
+    byName: (name, itemSearch = false) => {
+      cy.getElemByDataId(
+        itemSearch ? 'item-filter' : 'list-page-search-input',
+        timeouts.pageLoad,
+      ).type(name);
+    },
+  },
+  rows: {
+    countShouldBe: (count) => {
+      cy.get(`tr .co-resource-item__resource-name`).should('have.length', count);
```

Comment:
We really need `data-` attributes here. Let's avoid selectors like these (here and below).

Reply:
removed classname selectors